dev-qt/qt5/qt5-5.6.1.recipe

```diff
@@ -9,7 +9,7 @@ LICENSE="GNU LGPL v2"
 
 SOURCE_URI="https://download.qt.io/official_releases/qt/5.6/$portVersion/single/qt-everywhere-opensource-src-$portVersion.tar.xz"
```

Comment:
Good catch!
Wait, the SHA256 was correct... for the qt-everywhere-opensource-src-5.6.1.tar.**gz** ;-)

So if you could revert the CHECKSUM_SHA256 and replace the .tar.**xz** by a .tar.**gz** in SOURCE_URI that would be great!

BTW, while at it, would you mind removing the whitespace at the end of line 327 (which starts with devCommands) ? (I know it was already there before your edit, of course.)
Thanks !